oic.wk.atomicmeasurement.raml

```diff
@@ -0,0 +1,178 @@
+#%RAML 0.8
+#RAML for Atomic Measurement.
+#
+title: Atomic Measurement
+version: 0.1
+documentation:
+
+  - title: OCF Copyright
+    content: Copyright (c) 2016, 2017, 2018 Open Connectivity Foundation, Inc. All rights reserved.
+  - title: OCF License
+    content: !include LICENSE.md
+  - title: OCF Disclaimer
+    content: !include DISCLAIMER.md
+  - title: Atomic Measurement
+    content: |
+      Atomic Measurement is a specialization of the collection pattern to insure
+      that the linked resources are only accessed together.
+
+      Atomic Measurement exposes properties only through the oic.if.b interface,
+      thus ensuring that all of the items are accessed "atomically" in the batch.
+
+  - title: Resource Design
+    content: |
+      The Atomic Measurement resource is specified as three Base URIs, one
+      for each interface which is supported by the collection.
+
+      These three URIs are defined as the same URI path, only distinguished
+      by the different query parameter traits.
+
+schemas:
+  - baseline: !include schemas/oic.collection-schema.json
+  - batch-retrieve: !include schemas/oic.collection.batch-retrieve-schema.json
+  - links: !include schemas/oic.collection.linkslist-schema.json
+
+traits:
+  - interface-all:
+      queryParameters:
+        if:
+          enum: ["oic.if.b", "oic.if.ll", "oic.if.baseline"]
+
+  - interface-baseline:
+      queryParameters:
+        if:
+          enum: ["oic.if.baseline"]
+
+  - interface-b:
+      queryParameters:
+        if:
+          enum: ["oic.if.b"]
+
+  - interface-ll:
+      queryParameters:
+        if:
+          enum: ["oic.if.ll"]
+
+
+/AtomicMeasurementURI?if=oic.if.baseline:
+  description: |
+    The oic.if.baseline interface exposes a representation of the links and
+    the common properties of the Atomic Measurement resource.
+
+  displayName: Baseline Interface
+  is: [ interface-baseline ]
+
+  get:
+    description: |
+      Retrieve on Baseline Interface
+
+    responses:
+      200:
+        body:
+          application/json:
+            schema: baseline
+            example: |
+              {
+                "rt": ["oic.wk.col.atomic"],
+                "if": ["oic.if.b", "oic.if.ll", "oic.if.baseline"]
+                "rts": [ "oic.r.temperature", "oic.r.body.location.temperature", "oic.r.time.observed" ],
+                "links": [
+                  {
+                    "href": "temperature",
+                    "rt":   ["oic.r.temperature"],
+                    "if":   ["oic.if.s", "oic.if.baseline"],
+                  },
+                  {
+                    "href": "bodylocation",
+                    "rt":   ["oic.r.body.location.temperature"],
+                    "if":   ["oic.if.s", "oic.if.baseline"],
+                  },
+                  {
+                    "href": "timestamp",
+                    "rt":   ["oic.r.time.observed"],
```

Comment:
I can't find a schema for this but I did find one for oic.r.time.stamp.  Did that new schema replace this one?